README.md

```diff
@@ -10,61 +10,30 @@
 [![MongoDB Logo](https://img.shields.io/badge/-MongoDB-47A248?logo=MongoDB&logoColor=fff)][mongodb]
 [![Fastify](https://img.shields.io/badge/-Fastify.io-000000?logo=Fastify&logoColor=fff)][fastify]
 
-**Amibot** is a simple but robust bot that can access the contents from Amity Student Portal - *Amizone* and display them right in WhatsApp itself.
+Tired of slow and unreliable access to your Amity University student portal? Introducing Amibot, the chat bot that allows you to easily access all your important information on WhatsApp. With Amibot, you can quickly view your grades, attendance, class schedule, and more without ever having to log in to the Amizone website.
 
----
+## Features
 
-## What were the various issues which were being faced by the students ?
+- **Interactive and easy-to-use interface:** Amibot provides a simple and intuitive interface on WhatsApp, allowing you to access your information with just a few taps.
+- **Access to attendance and class schedule:** Amibot already provides information on your attendance and class schedule, and we're working on adding more features.
```

Comment:
**Access to attendance and class schedule:** Amibot provides information about your attendance and class schedule, and we're also working on adding more features.